### PR TITLE
Update audit-log-requirements.md

### PR DESCRIPTION
**Pull request by lbdvt**
Prebid SSO -> Prebid Addressability Framework

### CODE REVIEW

**Comment by OlivierChirouze** [comment]
Sorry for being annoying, but I have the feeling most of the replacements in this doc would sound better with "PAF".
WDYT?

File: mvp-spec/audit-log-requirements.md [hunk 1]
```diff
@@ -43,39 +43,39 @@ Model Terms and “**Parties**” shall be construed accordingly.
...
 the Receiver.
 
 **Sign** means a cryptographic confirmation of generating, sending or 
-receipt of Prebid SSO Data.
+receipt of Prebid Addressability Framework Data.
 
-**Transaction**: the sending of Prebid SSO Data from the Root Party through the 
-Prebid SSO ecosystem by consecutive Transmissions.
+**Transaction**: the sending of Prebid Addressability Framework Data from the Root Party through the 
+Prebid Addressability Framework ecosystem by consecutive Transmissions.
```
Comment (OlivierChirouze) [inline mvp-spec/audit-log-requirements.md:67]: I suggest: PAF instead

File: mvp-spec/audit-log-requirements.md [hunk 2]
```diff
@@ -98,18 +98,18 @@ formal complaint through the law enforcement mechanism of the relevant
...
 Those Audit Logs are available at least as long as the ads are shown. 
-More generally, a Prebid SSO ad is a Addressable Content.
+More generally, a Prebid Addressability Framework ad is a Addressable Content.
 
 ![Audit Button and Audit UI](./assets/audit-button-and-audit-ui-example.png)
 
 The Audit Log contains:
 
-* The Prebid SSO Data (Pseudonymous-identifiers + Preferences)
+* The Prebid Addressability Framework Data (Pseudonymous-identifiers + Preferences)
```
Comment (OlivierChirouze) [inline mvp-spec/audit-log-requirements.md:107]: PAF, etc.

**Review by OlivierChirouze — CHANGES_REQUESTED** [review]

**Comment by lbdvt** [comment]
You are probably right "PAF" is much shorter. 

What about:
- Renaming "Landscape" to "Introduction" (not sure if I can do that in a PR)
- Moving the "Definition" currently in "Audit log requirements" to "Introduction"
- Adding "PAF" definition as short for "Prebid Addressability Framework"
- Using "PAF" in most documents
?

**Comment by OlivierChirouze** [comment]
> Renaming "Landscape" to "Introduction" (not sure if I can do that in a PR)
It's part of my current work, please wait for it ;-) or give a first look [here](https://github.com/criteo/addressable-network-proposals/tree/entry-points/mvp-spec) if you are too impatient!
> Moving the "Definition" currently in "Audit log requirements" to "Introduction"
You're right, I'll try to include it in my PR or plan it for next step.

About using PAF, I tend to use the full naming in the first lines of my documents and then refer to it as "PAF". But we might indeed be very explicit in the main README.md file and then only use PAF everywhere else.

**Comment by lbdvt** [comment]
> About using PAF, I tend to use the full naming in the first lines of my documents and then refer to it as "PAF". But we might indeed be very explicit in the main README.md file and then only use PAF everywhere else.

Sounds good to me.

I'll drop this PR and create a new one to do just that in audit-log-requirements.md .

I'll let you do the work on "Introduction" file.

**Pull request closed without merging**